.github/workflows/release.yaml

```diff
@@ -2,7 +2,7 @@ name: Build and release Systemd sysext images
 on:
   push:
     tags:
-      - '*'
+      - 'latest'
```

Comment:
Improvement: Also release on a weekly cadence

Reply:
Ah, and should we always release `main`? Maybe a new action could run on every push and create a `latest` tag and trigger this action here (I think it won't get invoked by another workflow as this is a mechanism to prevent recursion).

Reply:
Eventually, yes. For this (and the weekly build cadence) to add value we should first make the build auto-detect new versions for all sysexts (like it does for Kubernetes already): I've opened a tracking issue for that: https://github.com/flatcar/sysext-bakery/issues/42